app/assets/javascripts/oxalis/view/tracing_layout_view.js

```diff
@@ -44,6 +45,12 @@ class TracingLayoutView extends React.PureComponent<Props, State> {
     });
   };
 
+  componentDidCatch() {
+    Toast.error(
+      "Unfortunateley, we encountered an error. We cannot guarantee that your work is persisted. Please reload the page and try again.",
```

Comment:
@hotzenklotz Apart from the "Unfortunateley" typo, do you have suggestions to improve this message, mr. copy master?

Reply:
Also, please move to messages.js